ERROR - 2025-02-02 00:23:25 --> mysqli_sql_exception: Table 'punto_venta.clientes' doesn't exist in C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\punto_venta\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\punto_venta\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\punto_venta\system\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\punto_venta\app\Controllers\Clientes.php(46): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(933): App\Controllers\Clientes->index()
#8 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
#9 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\punto_venta\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\punto_venta\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\punto_venta\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-02-02 00:23:25 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'punto_venta.clientes' doesn't exist
[Method: GET, Route: Clientes]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `clientes`
WHERE `activo` = :activo:', [...], false)
 2 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Clientes.php(46): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Clientes->index()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 00:23:25 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'punto_venta.clientes' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `clientes`
WHERE `activo` = 1')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `clientes`
WHERE `activo` = 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `clientes`
WHERE `activo` = :activo:', [...], false)
 4 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Controllers\Clientes.php(46): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Clientes->index()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 00:23:25 --> [Caused by] mysqli_sql_exception: Table 'punto_venta.clientes' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *
FROM `clientes`
WHERE `activo` = 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `clientes`
WHERE `activo` = 1')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `clientes`
WHERE `activo` = 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `clientes`
WHERE `activo` = :activo:', [...], false)
 5 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\Clientes.php(46): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Clientes->index()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-02-02 00:25:01 --> mysqli_sql_exception: Table 'punto_venta.clientes' doesn't exist in C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\punto_venta\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\punto_venta\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\punto_venta\system\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\punto_venta\app\Controllers\Clientes.php(46): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(933): App\Controllers\Clientes->index()
#8 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
#9 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\punto_venta\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\punto_venta\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\punto_venta\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-02-02 00:25:01 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'punto_venta.clientes' doesn't exist
[Method: GET, Route: Clientes]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `clientes`
WHERE `activo` = :activo:', [...], false)
 2 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Clientes.php(46): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Clientes->index()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 00:25:01 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'punto_venta.clientes' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `clientes`
WHERE `activo` = 1')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `clientes`
WHERE `activo` = 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `clientes`
WHERE `activo` = :activo:', [...], false)
 4 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Controllers\Clientes.php(46): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Clientes->index()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 00:25:01 --> [Caused by] mysqli_sql_exception: Table 'punto_venta.clientes' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *
FROM `clientes`
WHERE `activo` = 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `clientes`
WHERE `activo` = 1')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `clientes`
WHERE `activo` = 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `clientes`
WHERE `activo` = :activo:', [...], false)
 5 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\Clientes.php(46): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Clientes->index()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-02-02 00:25:22 --> mysqli_sql_exception: Table 'punto_venta.clientes' doesn't exist in C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\punto_venta\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\punto_venta\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\punto_venta\system\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\punto_venta\app\Controllers\Clientes.php(46): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(933): App\Controllers\Clientes->index()
#8 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
#9 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\punto_venta\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\punto_venta\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\punto_venta\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-02-02 00:25:22 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'punto_venta.clientes' doesn't exist
[Method: GET, Route: clientes]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `clientes`
WHERE `activo` = :activo:', [...], false)
 2 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Clientes.php(46): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Clientes->index()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 00:25:22 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'punto_venta.clientes' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `clientes`
WHERE `activo` = 1')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `clientes`
WHERE `activo` = 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `clientes`
WHERE `activo` = :activo:', [...], false)
 4 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Controllers\Clientes.php(46): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Clientes->index()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 00:25:22 --> [Caused by] mysqli_sql_exception: Table 'punto_venta.clientes' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *
FROM `clientes`
WHERE `activo` = 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `clientes`
WHERE `activo` = 1')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `clientes`
WHERE `activo` = 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `clientes`
WHERE `activo` = :activo:', [...], false)
 5 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\Clientes.php(46): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Clientes->index()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-02-02 00:25:29 --> mysqli_sql_exception: Table 'punto_venta.clientes' doesn't exist in C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\punto_venta\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\punto_venta\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\punto_venta\system\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\punto_venta\app\Controllers\Clientes.php(46): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(933): App\Controllers\Clientes->index()
#8 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
#9 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\punto_venta\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\punto_venta\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\punto_venta\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-02-02 00:25:29 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'punto_venta.clientes' doesn't exist
[Method: GET, Route: Clientes]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `clientes`
WHERE `activo` = :activo:', [...], false)
 2 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Clientes.php(46): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Clientes->index()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 00:25:29 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'punto_venta.clientes' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `clientes`
WHERE `activo` = 1')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `clientes`
WHERE `activo` = 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `clientes`
WHERE `activo` = :activo:', [...], false)
 4 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Controllers\Clientes.php(46): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Clientes->index()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 00:25:29 --> [Caused by] mysqli_sql_exception: Table 'punto_venta.clientes' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *
FROM `clientes`
WHERE `activo` = 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `clientes`
WHERE `activo` = 1')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `clientes`
WHERE `activo` = 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `clientes`
WHERE `activo` = :activo:', [...], false)
 5 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\Clientes.php(46): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Clientes->index()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 00:35:56 --> Error: Call to a member function where() on null
[Method: GET, Route: clientes/nuevo]
in APPPATH\Controllers\Clientes.php on line 64.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Clientes->nuevo()
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 00:40:54 --> Error: Call to a member function where() on null
[Method: GET, Route: clientes/nuevo]
in APPPATH\Controllers\Clientes.php on line 64.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Clientes->nuevo()
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 00:45:34 --> Error: Call to a member function where() on null
[Method: GET, Route: clientes/editar/1]
in APPPATH\Controllers\Clientes.php on line 85.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Clientes->editar('1')
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 01:31:19 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "configuracion/configuracion.php"
[Method: GET, Route: configuracion]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(212): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('configuracion/configuracion.php')
 2 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('configuracion/configuracion', [], true)
 3 APPPATH\Controllers\Configuracion.php(41): view('configuracion/configuracion')
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->index()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 01:33:17 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "configuracion/configuracion.php"
[Method: GET, Route: configuracion]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(212): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('configuracion/configuracion.php')
 2 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('configuracion/configuracion', [], true)
 3 APPPATH\Controllers\Configuracion.php(41): view('configuracion/configuracion')
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->index()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 01:47:34 --> ArgumentCountError: Too few arguments to function CodeIgniter\Database\BaseBuilder::where(), 0 passed in C:\xampp\htdocs\punto_venta\system\Model.php on line 928 and at least 1 expected
[Method: GET, Route: configuracion]
in SYSTEMPATH\Database\BaseBuilder.php on line 720.
 1 SYSTEMPATH\Model.php(928): CodeIgniter\Database\BaseBuilder->where()
 2 APPPATH\Controllers\Configuracion.php(37): CodeIgniter\Model->__call('where', [])
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->index()
 4 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
 5 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 01:51:07 --> ArgumentCountError: Too few arguments to function CodeIgniter\Database\BaseBuilder::where(), 0 passed in C:\xampp\htdocs\punto_venta\system\Model.php on line 928 and at least 1 expected
[Method: GET, Route: configuracion]
in SYSTEMPATH\Database\BaseBuilder.php on line 720.
 1 SYSTEMPATH\Model.php(928): CodeIgniter\Database\BaseBuilder->where()
 2 APPPATH\Controllers\Configuracion.php(37): CodeIgniter\Model->__call('where', [])
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->index()
 4 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
 5 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 01:51:10 --> ArgumentCountError: Too few arguments to function CodeIgniter\Database\BaseBuilder::where(), 0 passed in C:\xampp\htdocs\punto_venta\system\Model.php on line 928 and at least 1 expected
[Method: GET, Route: configuracion]
in SYSTEMPATH\Database\BaseBuilder.php on line 720.
 1 SYSTEMPATH\Model.php(928): CodeIgniter\Database\BaseBuilder->where()
 2 APPPATH\Controllers\Configuracion.php(37): CodeIgniter\Model->__call('where', [])
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->index()
 4 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
 5 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 01:51:19 --> ArgumentCountError: Too few arguments to function CodeIgniter\Database\BaseBuilder::where(), 0 passed in C:\xampp\htdocs\punto_venta\system\Model.php on line 928 and at least 1 expected
[Method: GET, Route: configuracion]
in SYSTEMPATH\Database\BaseBuilder.php on line 720.
 1 SYSTEMPATH\Model.php(928): CodeIgniter\Database\BaseBuilder->where()
 2 APPPATH\Controllers\Configuracion.php(37): CodeIgniter\Model->__call('where', [])
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->index()
 4 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
 5 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 01:53:33 --> ArgumentCountError: Too few arguments to function CodeIgniter\Database\BaseBuilder::where(), 0 passed in C:\xampp\htdocs\punto_venta\system\Model.php on line 928 and at least 1 expected
[Method: GET, Route: configuracion]
in SYSTEMPATH\Database\BaseBuilder.php on line 720.
 1 SYSTEMPATH\Model.php(928): CodeIgniter\Database\BaseBuilder->where()
 2 APPPATH\Controllers\Configuracion.php(37): CodeIgniter\Model->__call('where', [])
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->index()
 4 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
 5 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-02-02 02:14:10 --> mysqli_sql_exception: Unknown column 'fecha_edit' in 'field list' in C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `configu...', 0)
#1 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configu...')
#2 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configu...')
#3 C:\xampp\htdocs\punto_venta\system\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configu...', Array, false)
#4 C:\xampp\htdocs\punto_venta\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\punto_venta\system\BaseModel.php(1027): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\xampp\htdocs\punto_venta\system\Model.php(863): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\xampp\htdocs\punto_venta\app\Controllers\Configuracion.php(55): CodeIgniter\Model->update(Array, Array)
#8 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
#9 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
#10 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\punto_venta\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\punto_venta\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\punto_venta\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-02-02 02:14:10 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'fecha_edit' in 'field list'
[Method: POST, Route: configuracion/actualizar]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:, `fecha_edit` = :fecha_edit:
WHERE `configuracion`.`id` IN :configuracion.id:', [...], false)
 2 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate([...], [...])
 4 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update([...], [...])
 5 APPPATH\Controllers\Configuracion.php(55): CodeIgniter\Model->update([...], [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 02:14:10 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'fecha_edit' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configuracion` SET `valor` = \'Mi punto de venta1\', `fecha_edit` = \'2025-02-02 02:14:10\'
WHERE `configuracion`.`id` IN (\'tienda_nombre\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configuracion` SET `valor` = \'Mi punto de venta1\', `fecha_edit` = \'2025-02-02 02:14:10\'
WHERE `configuracion`.`id` IN (\'tienda_nombre\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:, `fecha_edit` = :fecha_edit:
WHERE `configuracion`.`id` IN :configuracion.id:', [...], false)
 4 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 5 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate([...], [...])
 6 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update([...], [...])
 7 APPPATH\Controllers\Configuracion.php(55): CodeIgniter\Model->update([...], [...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 02:14:10 --> [Caused by] mysqli_sql_exception: Unknown column 'fecha_edit' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `configuracion` SET `valor` = \'Mi punto de venta1\', `fecha_edit` = \'2025-02-02 02:14:10\'
WHERE `configuracion`.`id` IN (\'tienda_nombre\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configuracion` SET `valor` = \'Mi punto de venta1\', `fecha_edit` = \'2025-02-02 02:14:10\'
WHERE `configuracion`.`id` IN (\'tienda_nombre\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configuracion` SET `valor` = \'Mi punto de venta1\', `fecha_edit` = \'2025-02-02 02:14:10\'
WHERE `configuracion`.`id` IN (\'tienda_nombre\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:, `fecha_edit` = :fecha_edit:
WHERE `configuracion`.`id` IN :configuracion.id:', [...], false)
 5 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 6 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate([...], [...])
 7 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update([...], [...])
 8 APPPATH\Controllers\Configuracion.php(55): CodeIgniter\Model->update([...], [...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-02-02 02:20:10 --> mysqli_sql_exception: Unknown column 'fecha_edit' in 'field list' in C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `configu...', 0)
#1 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configu...')
#2 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configu...')
#3 C:\xampp\htdocs\punto_venta\system\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configu...', Array, false)
#4 C:\xampp\htdocs\punto_venta\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\punto_venta\system\BaseModel.php(1027): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\punto_venta\system\Model.php(863): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\punto_venta\app\Controllers\Configuracion.php(55): CodeIgniter\Model->update()
#8 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
#9 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
#10 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\punto_venta\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\punto_venta\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\punto_venta\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-02-02 02:20:10 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'fecha_edit' in 'field list'
[Method: POST, Route: configuracion/actualizar]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:, `fecha_edit` = :fecha_edit:
WHERE `nombre` IN :nombre:', [...], false)
 2 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 4 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 5 APPPATH\Controllers\Configuracion.php(55): CodeIgniter\Model->update()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 02:20:10 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'fecha_edit' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\', `fecha_edit` = \'2025-02-02 02:20:10\'
WHERE `nombre` IN (\'tienda_nombre\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\', `fecha_edit` = \'2025-02-02 02:20:10\'
WHERE `nombre` IN (\'tienda_nombre\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:, `fecha_edit` = :fecha_edit:
WHERE `nombre` IN :nombre:', [...], false)
 4 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 5 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 6 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 7 APPPATH\Controllers\Configuracion.php(55): CodeIgniter\Model->update()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 02:20:10 --> [Caused by] mysqli_sql_exception: Unknown column 'fecha_edit' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\', `fecha_edit` = \'2025-02-02 02:20:10\'
WHERE `nombre` IN (\'tienda_nombre\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\', `fecha_edit` = \'2025-02-02 02:20:10\'
WHERE `nombre` IN (\'tienda_nombre\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\', `fecha_edit` = \'2025-02-02 02:20:10\'
WHERE `nombre` IN (\'tienda_nombre\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:, `fecha_edit` = :fecha_edit:
WHERE `nombre` IN :nombre:', [...], false)
 5 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 6 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 7 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 8 APPPATH\Controllers\Configuracion.php(55): CodeIgniter\Model->update()
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-02-02 02:20:47 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:20:47'
WHERE `nombre` IN ('tienda_nombre')' at line 1 in C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `configu...', 0)
#1 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configu...')
#2 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configu...')
#3 C:\xampp\htdocs\punto_venta\system\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configu...', Array, false)
#4 C:\xampp\htdocs\punto_venta\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\punto_venta\system\BaseModel.php(1027): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\punto_venta\system\Model.php(863): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\punto_venta\app\Controllers\Configuracion.php(55): CodeIgniter\Model->update()
#8 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
#9 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
#10 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\punto_venta\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\punto_venta\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\punto_venta\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-02-02 02:20:47 --> CodeIgniter\Database\Exceptions\DatabaseException: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:20:47'
WHERE `nombre` IN ('tienda_nombre')' at line 1
[Method: POST, Route: configuracion/actualizar]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:,  = ::
WHERE `nombre` IN :nombre:', [...], false)
 2 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 4 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 5 APPPATH\Controllers\Configuracion.php(55): CodeIgniter\Model->update()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 02:20:47 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:20:47'
WHERE `nombre` IN ('tienda_nombre')' at line 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:20:47\'
WHERE `nombre` IN (\'tienda_nombre\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:20:47\'
WHERE `nombre` IN (\'tienda_nombre\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:,  = ::
WHERE `nombre` IN :nombre:', [...], false)
 4 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 5 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 6 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 7 APPPATH\Controllers\Configuracion.php(55): CodeIgniter\Model->update()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 02:20:47 --> [Caused by] mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:20:47'
WHERE `nombre` IN ('tienda_nombre')' at line 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:20:47\'
WHERE `nombre` IN (\'tienda_nombre\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:20:47\'
WHERE `nombre` IN (\'tienda_nombre\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:20:47\'
WHERE `nombre` IN (\'tienda_nombre\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:,  = ::
WHERE `nombre` IN :nombre:', [...], false)
 5 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 6 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 7 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 8 APPPATH\Controllers\Configuracion.php(55): CodeIgniter\Model->update()
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-02-02 02:20:53 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:20:53'
WHERE `nombre` IN ('tienda_nombre')' at line 1 in C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `configu...', 0)
#1 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configu...')
#2 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configu...')
#3 C:\xampp\htdocs\punto_venta\system\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configu...', Array, false)
#4 C:\xampp\htdocs\punto_venta\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\punto_venta\system\BaseModel.php(1027): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\punto_venta\system\Model.php(863): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\punto_venta\app\Controllers\Configuracion.php(55): CodeIgniter\Model->update()
#8 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
#9 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
#10 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\punto_venta\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\punto_venta\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\punto_venta\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-02-02 02:20:53 --> CodeIgniter\Database\Exceptions\DatabaseException: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:20:53'
WHERE `nombre` IN ('tienda_nombre')' at line 1
[Method: POST, Route: configuracion/actualizar]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:,  = ::
WHERE `nombre` IN :nombre:', [...], false)
 2 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 4 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 5 APPPATH\Controllers\Configuracion.php(55): CodeIgniter\Model->update()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 02:20:53 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:20:53'
WHERE `nombre` IN ('tienda_nombre')' at line 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:20:53\'
WHERE `nombre` IN (\'tienda_nombre\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:20:53\'
WHERE `nombre` IN (\'tienda_nombre\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:,  = ::
WHERE `nombre` IN :nombre:', [...], false)
 4 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 5 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 6 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 7 APPPATH\Controllers\Configuracion.php(55): CodeIgniter\Model->update()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 02:20:53 --> [Caused by] mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:20:53'
WHERE `nombre` IN ('tienda_nombre')' at line 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:20:53\'
WHERE `nombre` IN (\'tienda_nombre\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:20:53\'
WHERE `nombre` IN (\'tienda_nombre\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:20:53\'
WHERE `nombre` IN (\'tienda_nombre\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:,  = ::
WHERE `nombre` IN :nombre:', [...], false)
 5 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 6 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 7 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 8 APPPATH\Controllers\Configuracion.php(55): CodeIgniter\Model->update()
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-02-02 02:21:31 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:21:31'
WHERE `nombre` IN ('tienda_nombre')' at line 1 in C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `configu...', 0)
#1 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configu...')
#2 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configu...')
#3 C:\xampp\htdocs\punto_venta\system\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configu...', Array, false)
#4 C:\xampp\htdocs\punto_venta\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\punto_venta\system\BaseModel.php(1027): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\punto_venta\system\Model.php(863): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\punto_venta\app\Controllers\Configuracion.php(55): CodeIgniter\Model->update()
#8 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
#9 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
#10 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\punto_venta\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\punto_venta\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\punto_venta\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-02-02 02:21:31 --> CodeIgniter\Database\Exceptions\DatabaseException: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:21:31'
WHERE `nombre` IN ('tienda_nombre')' at line 1
[Method: POST, Route: configuracion/actualizar]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:,  = ::
WHERE `nombre` IN :nombre:', [...], false)
 2 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 4 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 5 APPPATH\Controllers\Configuracion.php(55): CodeIgniter\Model->update()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 02:21:31 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:21:31'
WHERE `nombre` IN ('tienda_nombre')' at line 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:21:31\'
WHERE `nombre` IN (\'tienda_nombre\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:21:31\'
WHERE `nombre` IN (\'tienda_nombre\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:,  = ::
WHERE `nombre` IN :nombre:', [...], false)
 4 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 5 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 6 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 7 APPPATH\Controllers\Configuracion.php(55): CodeIgniter\Model->update()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 02:21:31 --> [Caused by] mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:21:31'
WHERE `nombre` IN ('tienda_nombre')' at line 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:21:31\'
WHERE `nombre` IN (\'tienda_nombre\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:21:31\'
WHERE `nombre` IN (\'tienda_nombre\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:21:31\'
WHERE `nombre` IN (\'tienda_nombre\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:,  = ::
WHERE `nombre` IN :nombre:', [...], false)
 5 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 6 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 7 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 8 APPPATH\Controllers\Configuracion.php(55): CodeIgniter\Model->update()
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-02-02 02:25:27 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:25:27'
WHERE `nombre` = 'tienda_nombre'' at line 1 in C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `configu...', 0)
#1 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configu...')
#2 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configu...')
#3 C:\xampp\htdocs\punto_venta\system\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configu...', Array, false)
#4 C:\xampp\htdocs\punto_venta\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\punto_venta\system\BaseModel.php(1027): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\punto_venta\system\Model.php(863): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\punto_venta\app\Controllers\Configuracion.php(55): CodeIgniter\Model->update()
#8 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
#9 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
#10 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\punto_venta\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\punto_venta\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\punto_venta\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-02-02 02:25:27 --> CodeIgniter\Database\Exceptions\DatabaseException: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:25:27'
WHERE `nombre` = 'tienda_nombre'' at line 1
[Method: POST, Route: configuracion/actualizar]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:,  = ::
WHERE `nombre` = :nombre:', [...], false)
 2 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 4 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 5 APPPATH\Controllers\Configuracion.php(55): CodeIgniter\Model->update()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 02:25:27 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:25:27'
WHERE `nombre` = 'tienda_nombre'' at line 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:25:27\'
WHERE `nombre` = \'tienda_nombre\'')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:25:27\'
WHERE `nombre` = \'tienda_nombre\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:,  = ::
WHERE `nombre` = :nombre:', [...], false)
 4 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 5 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 6 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 7 APPPATH\Controllers\Configuracion.php(55): CodeIgniter\Model->update()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 02:25:27 --> [Caused by] mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:25:27'
WHERE `nombre` = 'tienda_nombre'' at line 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:25:27\'
WHERE `nombre` = \'tienda_nombre\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:25:27\'
WHERE `nombre` = \'tienda_nombre\'')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:25:27\'
WHERE `nombre` = \'tienda_nombre\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:,  = ::
WHERE `nombre` = :nombre:', [...], false)
 5 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 6 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 7 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 8 APPPATH\Controllers\Configuracion.php(55): CodeIgniter\Model->update()
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 02:25:38 --> CodeIgniter\Exceptions\InvalidArgumentException: whereIn() expects $values to be of type array or closure
[Method: POST, Route: configuracion/actualizar]
in SYSTEMPATH\Database\BaseBuilder.php on line 964.
 1 SYSTEMPATH\Database\BaseBuilder.php(850): CodeIgniter\Database\BaseBuilder->_whereIn('nombre', 'tienda_nombre', false, 'AND ', null)
 2 SYSTEMPATH\Model.php(928): CodeIgniter\Database\BaseBuilder->whereIn('nombre', 'tienda_nombre')
 3 APPPATH\Controllers\Configuracion.php(55): CodeIgniter\Model->__call('whereIn', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 02:26:32 --> CodeIgniter\Exceptions\InvalidArgumentException: whereIn() expects $values to be of type array or closure
[Method: POST, Route: configuracion/actualizar]
in SYSTEMPATH\Database\BaseBuilder.php on line 964.
 1 SYSTEMPATH\Database\BaseBuilder.php(850): CodeIgniter\Database\BaseBuilder->_whereIn('nombre', 'tienda_nombre', false, 'AND ', null)
 2 SYSTEMPATH\Model.php(928): CodeIgniter\Database\BaseBuilder->whereIn('nombre', 'tienda_nombre')
 3 APPPATH\Controllers\Configuracion.php(55): CodeIgniter\Model->__call('whereIn', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-02-02 02:27:50 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:27:50'
WHERE `nombre` = 'tienda_nombre'' at line 1 in C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `configu...', 0)
#1 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configu...')
#2 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configu...')
#3 C:\xampp\htdocs\punto_venta\system\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configu...', Array, false)
#4 C:\xampp\htdocs\punto_venta\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\punto_venta\system\BaseModel.php(1027): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\punto_venta\system\Model.php(863): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\punto_venta\app\Controllers\Configuracion.php(55): CodeIgniter\Model->update()
#8 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
#9 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
#10 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\punto_venta\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\punto_venta\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\punto_venta\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-02-02 02:27:50 --> CodeIgniter\Database\Exceptions\DatabaseException: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:27:50'
WHERE `nombre` = 'tienda_nombre'' at line 1
[Method: POST, Route: configuracion/actualizar]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:,  = ::
WHERE `nombre` = :nombre:', [...], false)
 2 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 4 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 5 APPPATH\Controllers\Configuracion.php(55): CodeIgniter\Model->update()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 02:27:50 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:27:50'
WHERE `nombre` = 'tienda_nombre'' at line 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:27:50\'
WHERE `nombre` = \'tienda_nombre\'')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:27:50\'
WHERE `nombre` = \'tienda_nombre\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:,  = ::
WHERE `nombre` = :nombre:', [...], false)
 4 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 5 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 6 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 7 APPPATH\Controllers\Configuracion.php(55): CodeIgniter\Model->update()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 02:27:50 --> [Caused by] mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:27:50'
WHERE `nombre` = 'tienda_nombre'' at line 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:27:50\'
WHERE `nombre` = \'tienda_nombre\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:27:50\'
WHERE `nombre` = \'tienda_nombre\'')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:27:50\'
WHERE `nombre` = \'tienda_nombre\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:,  = ::
WHERE `nombre` = :nombre:', [...], false)
 5 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 6 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 7 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 8 APPPATH\Controllers\Configuracion.php(55): CodeIgniter\Model->update()
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-02-02 02:28:34 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:28:34'
WHERE `nombre` = 'tienda_nombre'' at line 1 in C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `configu...', 0)
#1 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configu...')
#2 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configu...')
#3 C:\xampp\htdocs\punto_venta\system\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configu...', Array, false)
#4 C:\xampp\htdocs\punto_venta\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\punto_venta\system\BaseModel.php(1027): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\punto_venta\system\Model.php(863): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\punto_venta\app\Controllers\Configuracion.php(58): CodeIgniter\Model->update()
#8 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
#9 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
#10 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\punto_venta\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\punto_venta\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\punto_venta\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-02-02 02:28:34 --> CodeIgniter\Database\Exceptions\DatabaseException: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:28:34'
WHERE `nombre` = 'tienda_nombre'' at line 1
[Method: POST, Route: configuracion/actualizar]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:,  = ::
WHERE `nombre` = :nombre:', [...], false)
 2 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 4 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 5 APPPATH\Controllers\Configuracion.php(58): CodeIgniter\Model->update()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 02:28:34 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:28:34'
WHERE `nombre` = 'tienda_nombre'' at line 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:28:34\'
WHERE `nombre` = \'tienda_nombre\'')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:28:34\'
WHERE `nombre` = \'tienda_nombre\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:,  = ::
WHERE `nombre` = :nombre:', [...], false)
 4 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 5 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 6 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 7 APPPATH\Controllers\Configuracion.php(58): CodeIgniter\Model->update()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 02:28:34 --> [Caused by] mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:28:34'
WHERE `nombre` = 'tienda_nombre'' at line 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:28:34\'
WHERE `nombre` = \'tienda_nombre\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:28:34\'
WHERE `nombre` = \'tienda_nombre\'')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:28:34\'
WHERE `nombre` = \'tienda_nombre\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:,  = ::
WHERE `nombre` = :nombre:', [...], false)
 5 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 6 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 7 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 8 APPPATH\Controllers\Configuracion.php(58): CodeIgniter\Model->update()
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-02-02 02:29:21 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'punto de venta 21,  = '2025-02-02 02:29:21'
WHERE `nombre` = 'tienda_nombre'' at line 1 in C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `configu...', 0)
#1 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configu...')
#2 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configu...')
#3 C:\xampp\htdocs\punto_venta\system\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configu...', Array, false)
#4 C:\xampp\htdocs\punto_venta\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\punto_venta\system\BaseModel.php(1027): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\punto_venta\system\Model.php(863): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\punto_venta\app\Controllers\Configuracion.php(61): CodeIgniter\Model->update()
#8 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
#9 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
#10 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\punto_venta\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\punto_venta\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\punto_venta\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-02-02 02:29:21 --> CodeIgniter\Database\Exceptions\DatabaseException: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'punto de venta 21,  = '2025-02-02 02:29:21'
WHERE `nombre` = 'tienda_nombre'' at line 1
[Method: POST, Route: configuracion/actualizar]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = Mi punto de venta 21,  = ::
WHERE `nombre` = :nombre:', [...], false)
 2 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 4 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 5 APPPATH\Controllers\Configuracion.php(61): CodeIgniter\Model->update()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 02:29:21 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'punto de venta 21,  = '2025-02-02 02:29:21'
WHERE `nombre` = 'tienda_nombre'' at line 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configuracion` SET `valor` = Mi punto de venta 21,  = \'2025-02-02 02:29:21\'
WHERE `nombre` = \'tienda_nombre\'')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configuracion` SET `valor` = Mi punto de venta 21,  = \'2025-02-02 02:29:21\'
WHERE `nombre` = \'tienda_nombre\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = Mi punto de venta 21,  = ::
WHERE `nombre` = :nombre:', [...], false)
 4 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 5 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 6 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 7 APPPATH\Controllers\Configuracion.php(61): CodeIgniter\Model->update()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 02:29:21 --> [Caused by] mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'punto de venta 21,  = '2025-02-02 02:29:21'
WHERE `nombre` = 'tienda_nombre'' at line 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `configuracion` SET `valor` = Mi punto de venta 21,  = \'2025-02-02 02:29:21\'
WHERE `nombre` = \'tienda_nombre\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configuracion` SET `valor` = Mi punto de venta 21,  = \'2025-02-02 02:29:21\'
WHERE `nombre` = \'tienda_nombre\'')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configuracion` SET `valor` = Mi punto de venta 21,  = \'2025-02-02 02:29:21\'
WHERE `nombre` = \'tienda_nombre\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = Mi punto de venta 21,  = ::
WHERE `nombre` = :nombre:', [...], false)
 5 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 6 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 7 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 8 APPPATH\Controllers\Configuracion.php(61): CodeIgniter\Model->update()
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-02-02 02:30:06 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:30:06'
WHERE `nombre` = 'tienda_nombre'' at line 1 in C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `configu...', 0)
#1 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configu...')
#2 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configu...')
#3 C:\xampp\htdocs\punto_venta\system\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configu...', Array, false)
#4 C:\xampp\htdocs\punto_venta\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\punto_venta\system\BaseModel.php(1027): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\punto_venta\system\Model.php(863): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\punto_venta\app\Controllers\Configuracion.php(60): CodeIgniter\Model->update()
#8 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
#9 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
#10 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\punto_venta\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\punto_venta\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\punto_venta\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-02-02 02:30:06 --> CodeIgniter\Database\Exceptions\DatabaseException: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:30:06'
WHERE `nombre` = 'tienda_nombre'' at line 1
[Method: POST, Route: configuracion/actualizar]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:,  = ::
WHERE `nombre` = :nombre:', [...], false)
 2 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 4 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 5 APPPATH\Controllers\Configuracion.php(60): CodeIgniter\Model->update()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 02:30:06 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:30:06'
WHERE `nombre` = 'tienda_nombre'' at line 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:30:06\'
WHERE `nombre` = \'tienda_nombre\'')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:30:06\'
WHERE `nombre` = \'tienda_nombre\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:,  = ::
WHERE `nombre` = :nombre:', [...], false)
 4 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 5 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 6 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 7 APPPATH\Controllers\Configuracion.php(60): CodeIgniter\Model->update()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 02:30:06 --> [Caused by] mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:30:06'
WHERE `nombre` = 'tienda_nombre'' at line 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:30:06\'
WHERE `nombre` = \'tienda_nombre\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:30:06\'
WHERE `nombre` = \'tienda_nombre\'')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:30:06\'
WHERE `nombre` = \'tienda_nombre\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:,  = ::
WHERE `nombre` = :nombre:', [...], false)
 5 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 6 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 7 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 8 APPPATH\Controllers\Configuracion.php(60): CodeIgniter\Model->update()
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-02-02 02:31:02 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:31:02'
WHERE `nombre` = 'tienda_nombre'' at line 1 in C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `configu...', 0)
#1 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configu...')
#2 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configu...')
#3 C:\xampp\htdocs\punto_venta\system\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configu...', Array, false)
#4 C:\xampp\htdocs\punto_venta\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\punto_venta\system\BaseModel.php(1027): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\punto_venta\system\Model.php(863): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\punto_venta\app\Controllers\Configuracion.php(60): CodeIgniter\Model->update()
#8 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
#9 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
#10 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\punto_venta\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\punto_venta\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\punto_venta\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-02-02 02:31:02 --> CodeIgniter\Database\Exceptions\DatabaseException: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:31:02'
WHERE `nombre` = 'tienda_nombre'' at line 1
[Method: POST, Route: configuracion/actualizar]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:,  = ::
WHERE `nombre` = :nombre:', [...], false)
 2 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 4 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 5 APPPATH\Controllers\Configuracion.php(60): CodeIgniter\Model->update()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 02:31:02 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:31:02'
WHERE `nombre` = 'tienda_nombre'' at line 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:31:02\'
WHERE `nombre` = \'tienda_nombre\'')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:31:02\'
WHERE `nombre` = \'tienda_nombre\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:,  = ::
WHERE `nombre` = :nombre:', [...], false)
 4 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 5 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 6 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 7 APPPATH\Controllers\Configuracion.php(60): CodeIgniter\Model->update()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 02:31:02 --> [Caused by] mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:31:02'
WHERE `nombre` = 'tienda_nombre'' at line 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:31:02\'
WHERE `nombre` = \'tienda_nombre\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:31:02\'
WHERE `nombre` = \'tienda_nombre\'')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:31:02\'
WHERE `nombre` = \'tienda_nombre\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:,  = ::
WHERE `nombre` = :nombre:', [...], false)
 5 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 6 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 7 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 8 APPPATH\Controllers\Configuracion.php(60): CodeIgniter\Model->update()
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-02-02 02:31:54 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:31:54'
WHERE `nombre` IN ('tienda_nombre')' at line 1 in C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `configu...', 0)
#1 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configu...')
#2 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configu...')
#3 C:\xampp\htdocs\punto_venta\system\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configu...', Array, false)
#4 C:\xampp\htdocs\punto_venta\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\punto_venta\system\BaseModel.php(1027): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\punto_venta\system\Model.php(863): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\punto_venta\app\Controllers\Configuracion.php(56): CodeIgniter\Model->update()
#8 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
#9 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
#10 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\punto_venta\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\punto_venta\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\punto_venta\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-02-02 02:31:54 --> CodeIgniter\Database\Exceptions\DatabaseException: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:31:54'
WHERE `nombre` IN ('tienda_nombre')' at line 1
[Method: POST, Route: configuracion/actualizar]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:,  = ::
WHERE `nombre` IN :nombre:', [...], false)
 2 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 4 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 5 APPPATH\Controllers\Configuracion.php(56): CodeIgniter\Model->update()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 02:31:54 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:31:54'
WHERE `nombre` IN ('tienda_nombre')' at line 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:31:54\'
WHERE `nombre` IN (\'tienda_nombre\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:31:54\'
WHERE `nombre` IN (\'tienda_nombre\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:,  = ::
WHERE `nombre` IN :nombre:', [...], false)
 4 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 5 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 6 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 7 APPPATH\Controllers\Configuracion.php(56): CodeIgniter\Model->update()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 02:31:54 --> [Caused by] mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:31:54'
WHERE `nombre` IN ('tienda_nombre')' at line 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:31:54\'
WHERE `nombre` IN (\'tienda_nombre\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:31:54\'
WHERE `nombre` IN (\'tienda_nombre\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:31:54\'
WHERE `nombre` IN (\'tienda_nombre\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:,  = ::
WHERE `nombre` IN :nombre:', [...], false)
 5 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 6 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 7 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 8 APPPATH\Controllers\Configuracion.php(56): CodeIgniter\Model->update()
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-02-02 02:31:59 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:31:59'
WHERE `nombre` IN ('tienda_nombre')' at line 1 in C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `configu...', 0)
#1 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configu...')
#2 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configu...')
#3 C:\xampp\htdocs\punto_venta\system\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configu...', Array, false)
#4 C:\xampp\htdocs\punto_venta\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\punto_venta\system\BaseModel.php(1027): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\punto_venta\system\Model.php(863): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\punto_venta\app\Controllers\Configuracion.php(56): CodeIgniter\Model->update()
#8 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
#9 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
#10 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\punto_venta\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\punto_venta\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\punto_venta\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-02-02 02:31:59 --> CodeIgniter\Database\Exceptions\DatabaseException: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:31:59'
WHERE `nombre` IN ('tienda_nombre')' at line 1
[Method: POST, Route: configuracion/actualizar]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:,  = ::
WHERE `nombre` IN :nombre:', [...], false)
 2 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 4 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 5 APPPATH\Controllers\Configuracion.php(56): CodeIgniter\Model->update()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 02:31:59 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:31:59'
WHERE `nombre` IN ('tienda_nombre')' at line 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 2\',  = \'2025-02-02 02:31:59\'
WHERE `nombre` IN (\'tienda_nombre\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 2\',  = \'2025-02-02 02:31:59\'
WHERE `nombre` IN (\'tienda_nombre\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:,  = ::
WHERE `nombre` IN :nombre:', [...], false)
 4 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 5 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 6 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 7 APPPATH\Controllers\Configuracion.php(56): CodeIgniter\Model->update()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 02:31:59 --> [Caused by] mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:31:59'
WHERE `nombre` IN ('tienda_nombre')' at line 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 2\',  = \'2025-02-02 02:31:59\'
WHERE `nombre` IN (\'tienda_nombre\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 2\',  = \'2025-02-02 02:31:59\'
WHERE `nombre` IN (\'tienda_nombre\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 2\',  = \'2025-02-02 02:31:59\'
WHERE `nombre` IN (\'tienda_nombre\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:,  = ::
WHERE `nombre` IN :nombre:', [...], false)
 5 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 6 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 7 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 8 APPPATH\Controllers\Configuracion.php(56): CodeIgniter\Model->update()
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-02-02 02:33:33 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:33:33'
WHERE `nombre` IN ('tienda_nombre')' at line 1 in C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `configu...', 0)
#1 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configu...')
#2 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configu...')
#3 C:\xampp\htdocs\punto_venta\system\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configu...', Array, false)
#4 C:\xampp\htdocs\punto_venta\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\punto_venta\system\BaseModel.php(1027): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\punto_venta\system\Model.php(863): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\punto_venta\app\Controllers\Configuracion.php(56): CodeIgniter\Model->update()
#8 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
#9 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
#10 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\punto_venta\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\punto_venta\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\punto_venta\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-02-02 02:33:33 --> CodeIgniter\Database\Exceptions\DatabaseException: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:33:33'
WHERE `nombre` IN ('tienda_nombre')' at line 1
[Method: POST, Route: configuracion/actualizar]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:,  = ::
WHERE `nombre` IN :nombre:', [...], false)
 2 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 4 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 5 APPPATH\Controllers\Configuracion.php(56): CodeIgniter\Model->update()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 02:33:33 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:33:33'
WHERE `nombre` IN ('tienda_nombre')' at line 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:33:33\'
WHERE `nombre` IN (\'tienda_nombre\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:33:33\'
WHERE `nombre` IN (\'tienda_nombre\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:,  = ::
WHERE `nombre` IN :nombre:', [...], false)
 4 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 5 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 6 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 7 APPPATH\Controllers\Configuracion.php(56): CodeIgniter\Model->update()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 02:33:33 --> [Caused by] mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:33:33'
WHERE `nombre` IN ('tienda_nombre')' at line 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:33:33\'
WHERE `nombre` IN (\'tienda_nombre\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:33:33\'
WHERE `nombre` IN (\'tienda_nombre\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:33:33\'
WHERE `nombre` IN (\'tienda_nombre\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:,  = ::
WHERE `nombre` IN :nombre:', [...], false)
 5 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 6 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 7 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 8 APPPATH\Controllers\Configuracion.php(56): CodeIgniter\Model->update()
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-02-02 02:36:28 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:36:28'
WHERE `nombre` IN ('tienda_nombre')' at line 1 in C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `configu...', 0)
#1 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configu...')
#2 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configu...')
#3 C:\xampp\htdocs\punto_venta\system\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configu...', Array, false)
#4 C:\xampp\htdocs\punto_venta\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\punto_venta\system\BaseModel.php(1027): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\punto_venta\system\Model.php(863): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\punto_venta\app\Controllers\Configuracion.php(56): CodeIgniter\Model->update()
#8 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
#9 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
#10 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\punto_venta\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\punto_venta\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\punto_venta\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-02-02 02:36:28 --> CodeIgniter\Database\Exceptions\DatabaseException: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:36:28'
WHERE `nombre` IN ('tienda_nombre')' at line 1
[Method: POST, Route: configuracion/actualizar]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:,  = ::
WHERE `nombre` IN :nombre:', [...], false)
 2 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 4 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 5 APPPATH\Controllers\Configuracion.php(56): CodeIgniter\Model->update()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 02:36:28 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:36:28'
WHERE `nombre` IN ('tienda_nombre')' at line 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:36:28\'
WHERE `nombre` IN (\'tienda_nombre\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:36:28\'
WHERE `nombre` IN (\'tienda_nombre\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:,  = ::
WHERE `nombre` IN :nombre:', [...], false)
 4 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 5 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 6 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 7 APPPATH\Controllers\Configuracion.php(56): CodeIgniter\Model->update()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 02:36:28 --> [Caused by] mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:36:28'
WHERE `nombre` IN ('tienda_nombre')' at line 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:36:28\'
WHERE `nombre` IN (\'tienda_nombre\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:36:28\'
WHERE `nombre` IN (\'tienda_nombre\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:36:28\'
WHERE `nombre` IN (\'tienda_nombre\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:,  = ::
WHERE `nombre` IN :nombre:', [...], false)
 5 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 6 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 7 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 8 APPPATH\Controllers\Configuracion.php(56): CodeIgniter\Model->update()
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-02-02 02:40:10 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:40:10'
WHERE `nombre` IN ('tienda_nombre')' at line 1 in C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `configu...', 0)
#1 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configu...')
#2 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configu...')
#3 C:\xampp\htdocs\punto_venta\system\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configu...', Array, false)
#4 C:\xampp\htdocs\punto_venta\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\punto_venta\system\BaseModel.php(1027): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\punto_venta\system\Model.php(863): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\punto_venta\app\Controllers\Configuracion.php(57): CodeIgniter\Model->update()
#8 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
#9 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
#10 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\punto_venta\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\punto_venta\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\punto_venta\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-02-02 02:40:10 --> CodeIgniter\Database\Exceptions\DatabaseException: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:40:10'
WHERE `nombre` IN ('tienda_nombre')' at line 1
[Method: POST, Route: configuracion/actualizar]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:,  = ::
WHERE `nombre` IN :nombre:', [...], false)
 2 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 4 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 5 APPPATH\Controllers\Configuracion.php(57): CodeIgniter\Model->update()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 02:40:10 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:40:10'
WHERE `nombre` IN ('tienda_nombre')' at line 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:40:10\'
WHERE `nombre` IN (\'tienda_nombre\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:40:10\'
WHERE `nombre` IN (\'tienda_nombre\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:,  = ::
WHERE `nombre` IN :nombre:', [...], false)
 4 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 5 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 6 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 7 APPPATH\Controllers\Configuracion.php(57): CodeIgniter\Model->update()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 02:40:10 --> [Caused by] mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:40:10'
WHERE `nombre` IN ('tienda_nombre')' at line 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:40:10\'
WHERE `nombre` IN (\'tienda_nombre\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:40:10\'
WHERE `nombre` IN (\'tienda_nombre\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:40:10\'
WHERE `nombre` IN (\'tienda_nombre\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:,  = ::
WHERE `nombre` IN :nombre:', [...], false)
 5 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 6 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 7 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 8 APPPATH\Controllers\Configuracion.php(57): CodeIgniter\Model->update()
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-02-02 02:42:39 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:42:39'
WHERE `nombre` IN ('tienda_nombre')' at line 1 in C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `configu...', 0)
#1 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configu...')
#2 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configu...')
#3 C:\xampp\htdocs\punto_venta\system\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configu...', Array, false)
#4 C:\xampp\htdocs\punto_venta\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\punto_venta\system\BaseModel.php(1027): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\punto_venta\system\Model.php(863): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\punto_venta\app\Controllers\Configuracion.php(60): CodeIgniter\Model->update()
#8 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
#9 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
#10 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\punto_venta\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\punto_venta\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\punto_venta\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
ERROR - 2025-02-02 02:42:39 --> You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:42:39'
WHERE `nombre` IN ('tienda_nombre')' at line 1
DEBUG - 2025-02-02 02:42:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-02-02 02:42:43 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:42:43'
WHERE `nombre` IN ('tienda_nombre')' at line 1 in C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `configu...', 0)
#1 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configu...')
#2 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configu...')
#3 C:\xampp\htdocs\punto_venta\system\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configu...', Array, false)
#4 C:\xampp\htdocs\punto_venta\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\punto_venta\system\BaseModel.php(1027): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\punto_venta\system\Model.php(863): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\punto_venta\app\Controllers\Configuracion.php(60): CodeIgniter\Model->update()
#8 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
#9 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
#10 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\punto_venta\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\punto_venta\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\punto_venta\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
ERROR - 2025-02-02 02:42:43 --> You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:42:43'
WHERE `nombre` IN ('tienda_nombre')' at line 1
DEBUG - 2025-02-02 02:42:43 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-02-02 02:43:05 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:43:05'
WHERE `nombre` IN ('tienda_nombre')' at line 1 in C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `configu...', 0)
#1 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configu...')
#2 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configu...')
#3 C:\xampp\htdocs\punto_venta\system\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configu...', Array, false)
#4 C:\xampp\htdocs\punto_venta\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\punto_venta\system\BaseModel.php(1027): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\punto_venta\system\Model.php(863): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\punto_venta\app\Controllers\Configuracion.php(60): CodeIgniter\Model->update()
#8 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
#9 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
#10 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\punto_venta\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\punto_venta\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\punto_venta\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
ERROR - 2025-02-02 02:43:05 --> You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:43:05'
WHERE `nombre` IN ('tienda_nombre')' at line 1
DEBUG - 2025-02-02 02:43:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-02-02 02:46:11 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:46:11'
WHERE `nombre` IN ('tienda_nombre')' at line 1 in C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `configu...', 0)
#1 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configu...')
#2 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configu...')
#3 C:\xampp\htdocs\punto_venta\system\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configu...', Array, false)
#4 C:\xampp\htdocs\punto_venta\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\punto_venta\system\BaseModel.php(1027): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\punto_venta\system\Model.php(863): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\punto_venta\app\Controllers\Configuracion.php(60): CodeIgniter\Model->update()
#8 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
#9 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
#10 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\punto_venta\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\punto_venta\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\punto_venta\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
ERROR - 2025-02-02 02:46:11 --> You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:46:11'
WHERE `nombre` IN ('tienda_nombre')' at line 1
DEBUG - 2025-02-02 02:46:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-02-02 02:53:14 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:53:14'
WHERE `nombre` IN ('tienda_nombre')' at line 1 in C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `configu...', 0)
#1 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configu...')
#2 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configu...')
#3 C:\xampp\htdocs\punto_venta\system\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configu...', Array, false)
#4 C:\xampp\htdocs\punto_venta\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\punto_venta\system\BaseModel.php(1027): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\punto_venta\system\Model.php(863): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\punto_venta\app\Controllers\Configuracion.php(57): CodeIgniter\Model->update()
#8 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
#9 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
#10 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\punto_venta\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\punto_venta\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\punto_venta\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-02-02 02:53:14 --> CodeIgniter\Database\Exceptions\DatabaseException: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:53:14'
WHERE `nombre` IN ('tienda_nombre')' at line 1
[Method: POST, Route: configuracion/actualizar]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:,  = ::
WHERE `nombre` IN :nombre:', [...], false)
 2 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 4 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 5 APPPATH\Controllers\Configuracion.php(57): CodeIgniter\Model->update()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 02:53:14 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:53:14'
WHERE `nombre` IN ('tienda_nombre')' at line 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:53:14\'
WHERE `nombre` IN (\'tienda_nombre\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:53:14\'
WHERE `nombre` IN (\'tienda_nombre\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:,  = ::
WHERE `nombre` IN :nombre:', [...], false)
 4 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 5 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 6 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 7 APPPATH\Controllers\Configuracion.php(57): CodeIgniter\Model->update()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 02:53:14 --> [Caused by] mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:53:14'
WHERE `nombre` IN ('tienda_nombre')' at line 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:53:14\'
WHERE `nombre` IN (\'tienda_nombre\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:53:14\'
WHERE `nombre` IN (\'tienda_nombre\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 02:53:14\'
WHERE `nombre` IN (\'tienda_nombre\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:,  = ::
WHERE `nombre` IN :nombre:', [...], false)
 5 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 6 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 7 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 8 APPPATH\Controllers\Configuracion.php(57): CodeIgniter\Model->update()
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-02-02 02:55:52 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:55:52'
WHERE `nombre` = 'tienda_nombre'' at line 1 in C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `configu...', 0)
#1 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configu...')
#2 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configu...')
#3 C:\xampp\htdocs\punto_venta\system\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configu...', Array, false)
#4 C:\xampp\htdocs\punto_venta\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\punto_venta\system\BaseModel.php(1027): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\punto_venta\system\Model.php(863): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\punto_venta\app\Controllers\Configuracion.php(63): CodeIgniter\Model->update()
#8 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
#9 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
#10 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\punto_venta\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\punto_venta\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\punto_venta\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
ERROR - 2025-02-02 02:55:52 --> You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:55:52'
WHERE `nombre` = 'tienda_nombre'' at line 1
DEBUG - 2025-02-02 02:55:52 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-02-02 02:55:58 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:55:58'
WHERE `nombre` = 'tienda_nombre'' at line 1 in C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `configu...', 0)
#1 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configu...')
#2 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configu...')
#3 C:\xampp\htdocs\punto_venta\system\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configu...', Array, false)
#4 C:\xampp\htdocs\punto_venta\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\punto_venta\system\BaseModel.php(1027): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\punto_venta\system\Model.php(863): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\punto_venta\app\Controllers\Configuracion.php(63): CodeIgniter\Model->update()
#8 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
#9 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
#10 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\punto_venta\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\punto_venta\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\punto_venta\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
ERROR - 2025-02-02 02:55:58 --> You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:55:58'
WHERE `nombre` = 'tienda_nombre'' at line 1
DEBUG - 2025-02-02 02:55:58 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-02-02 02:56:06 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:56:06'
WHERE `nombre` = 'tienda_nombre'' at line 1 in C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `configu...', 0)
#1 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configu...')
#2 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configu...')
#3 C:\xampp\htdocs\punto_venta\system\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configu...', Array, false)
#4 C:\xampp\htdocs\punto_venta\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\punto_venta\system\BaseModel.php(1027): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\punto_venta\system\Model.php(863): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\punto_venta\app\Controllers\Configuracion.php(63): CodeIgniter\Model->update()
#8 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
#9 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
#10 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\punto_venta\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\punto_venta\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\punto_venta\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
ERROR - 2025-02-02 02:56:06 --> You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:56:06'
WHERE `nombre` = 'tienda_nombre'' at line 1
DEBUG - 2025-02-02 02:56:06 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-02-02 02:57:42 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:57:42'
WHERE `nombre` = 'tienda_nombre'' at line 1 in C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `configu...', 0)
#1 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configu...')
#2 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configu...')
#3 C:\xampp\htdocs\punto_venta\system\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configu...', Array, false)
#4 C:\xampp\htdocs\punto_venta\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\punto_venta\system\BaseModel.php(1027): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\punto_venta\system\Model.php(863): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\punto_venta\app\Controllers\Configuracion.php(69): CodeIgniter\Model->update()
#8 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
#9 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
#10 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\punto_venta\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\punto_venta\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\punto_venta\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
ERROR - 2025-02-02 02:57:42 --> You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 02:57:42'
WHERE `nombre` = 'tienda_nombre'' at line 1
DEBUG - 2025-02-02 02:57:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-02-02 03:00:33 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 03:00:33'
WHERE `nombre` IN ('tienda_nombre')' at line 1 in C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `configu...', 0)
#1 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configu...')
#2 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configu...')
#3 C:\xampp\htdocs\punto_venta\system\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configu...', Array, false)
#4 C:\xampp\htdocs\punto_venta\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\punto_venta\system\BaseModel.php(1027): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\punto_venta\system\Model.php(863): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\punto_venta\app\Controllers\Configuracion.php(57): CodeIgniter\Model->update()
#8 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
#9 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
#10 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\punto_venta\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\punto_venta\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\punto_venta\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-02-02 03:00:33 --> CodeIgniter\Database\Exceptions\DatabaseException: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 03:00:33'
WHERE `nombre` IN ('tienda_nombre')' at line 1
[Method: POST, Route: configuracion/actualizar]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:,  = ::
WHERE `nombre` IN :nombre:', [...], false)
 2 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 4 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 5 APPPATH\Controllers\Configuracion.php(57): CodeIgniter\Model->update()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 03:00:33 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 03:00:33'
WHERE `nombre` IN ('tienda_nombre')' at line 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 03:00:33\'
WHERE `nombre` IN (\'tienda_nombre\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 03:00:33\'
WHERE `nombre` IN (\'tienda_nombre\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:,  = ::
WHERE `nombre` IN :nombre:', [...], false)
 4 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 5 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 6 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 7 APPPATH\Controllers\Configuracion.php(57): CodeIgniter\Model->update()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 03:00:33 --> [Caused by] mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '= '2025-02-02 03:00:33'
WHERE `nombre` IN ('tienda_nombre')' at line 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 03:00:33\'
WHERE `nombre` IN (\'tienda_nombre\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 03:00:33\'
WHERE `nombre` IN (\'tienda_nombre\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configuracion` SET `valor` = \'Mi punto de venta 1\',  = \'2025-02-02 03:00:33\'
WHERE `nombre` IN (\'tienda_nombre\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:,  = ::
WHERE `nombre` IN :nombre:', [...], false)
 5 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 6 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 7 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 8 APPPATH\Controllers\Configuracion.php(57): CodeIgniter\Model->update()
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-02-02 03:16:20 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\punto_venta\system\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `configu...', 0)
#1 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configu...')
#2 C:\xampp\htdocs\punto_venta\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configu...')
#3 C:\xampp\htdocs\punto_venta\system\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configu...', Array, false)
#4 C:\xampp\htdocs\punto_venta\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\punto_venta\system\BaseModel.php(1027): CodeIgniter\Model->doUpdate(NULL, Array)
#6 C:\xampp\htdocs\punto_venta\system\Model.php(863): CodeIgniter\BaseModel->update(NULL, Array)
#7 C:\xampp\htdocs\punto_venta\app\Controllers\Configuracion.php(55): CodeIgniter\Model->update()
#8 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
#9 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
#10 C:\xampp\htdocs\punto_venta\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\punto_venta\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\punto_venta\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\punto_venta\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-02-02 03:16:20 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'updated_at' in 'field list'
[Method: POST, Route: configuracion/actualizar]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:, `updated_at` = :updated_at:
WHERE `nombre` IN :nombre:', [...], false)
 2 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 4 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 5 APPPATH\Controllers\Configuracion.php(55): CodeIgniter\Model->update()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 03:16:20 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configuracion` SET `valor` = \'Fish House1\', `updated_at` = \'2025-02-02 03:16:20\'
WHERE `nombre` IN (\'tienda_nombre\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configuracion` SET `valor` = \'Fish House1\', `updated_at` = \'2025-02-02 03:16:20\'
WHERE `nombre` IN (\'tienda_nombre\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:, `updated_at` = :updated_at:
WHERE `nombre` IN :nombre:', [...], false)
 4 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 5 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 6 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 7 APPPATH\Controllers\Configuracion.php(55): CodeIgniter\Model->update()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-02 03:16:20 --> [Caused by] mysqli_sql_exception: Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `configuracion` SET `valor` = \'Fish House1\', `updated_at` = \'2025-02-02 03:16:20\'
WHERE `nombre` IN (\'tienda_nombre\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `configuracion` SET `valor` = \'Fish House1\', `updated_at` = \'2025-02-02 03:16:20\'
WHERE `nombre` IN (\'tienda_nombre\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `configuracion` SET `valor` = \'Fish House1\', `updated_at` = \'2025-02-02 03:16:20\'
WHERE `nombre` IN (\'tienda_nombre\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `configuracion` SET `valor` = :valor:, `updated_at` = :updated_at:
WHERE `nombre` IN :nombre:', [...], false)
 5 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 6 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate(null, [...])
 7 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update(null, [...])
 8 APPPATH\Controllers\Configuracion.php(55): CodeIgniter\Model->update()
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Configuracion->actualizar()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Configuracion))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
